rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -95,6 +98,33 @@ def fixture_accounting_initialize_parameters():
     return False
 
 
+@pytest.fixture(name='initialize_accounting_rules')
+def fixture_initialize_accounting_rules() -> bool:
+    return True
+
+
+@pytest.fixture(name='last_accounting_rules_version', scope='session')
+def fixture_last_accounting_rules_version() -> int:
+    return 1
+
+
+@pytest.fixture(name='latest_accounting_rules', autouse=True, scope='session')
+def fixture_download_rules(last_accounting_rules_version) -> Path:
+    root_dir = default_data_directory().parent / 'test-caching'
+    base_dir = root_dir / 'rules'
+    rules_file = Path(base_dir / f'v{last_accounting_rules_version}.json')
+
+    if rules_file.exists():
+        return rules_file
+
+    response = requests.get(f'https://raw.githubusercontent.com/rotki/data/develop/updates/accounting_rules/v{last_accounting_rules_version}.json')
+    rules_file.parent.mkdir(exist_ok=True, parents=True)
+    with open(rules_file, 'w+') as f:
```

Comment:
why 'w+'`?

Reply:
The `w+` is to create the file if it doesn't exists. Pathlib creates the path tree and then this the file if needed

Reply:
That's not true. 'w' will still create the file if it does not exist. In both cases old file will be erased. Difference is you can also read with this. In short you don't need this. You need `'w'`